### PR TITLE
Papilio 87 implement missing tests for business endpoints

### PR DESCRIPTION
**Pull request by Mikeymousey**
Worked through a series of bugs related to createBusiness endpoint e2e testing. 

1. Split server and app into two different files for better coding practices purpose but there is a big tradeoff with having to refactor the code. For time concerns, I decided not to include this change in this PR.
2. Kept getting BAD REQUEST error since the testBusiness token used in the tests did not follow schema.
3. Then started getting HTTP Status code 500 since createBusiness function implementation includes a call to the getBusinessById method which had to be spied for the test to properly function.

### CODE REVIEW

File: src/api/controllers/tests/user-controller.test.ts [hunk 1]
```diff
@@ -297,6 +300,12 @@ describe('UserController', () => {
         //    GETUSERACTIVITYLIST ENDPOINT    //
         ////////////////////////////////////////
         describe('getUserActivityList endpoint', () => {
+            afterEach(() => {
+                server.close();
+            });
+            afterAll(() => {
+                server.close();
+            });
```
Comment (mth-1012) [inline src/api/controllers/tests/user-controller.test.ts:308]: we probably don't need both

File: src/api/controllers/tests/business-controller.test.ts [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+        describe('removeEmployee endpoint', () => {
+            afterEach(() => {
+                server.close();
+            });
+            it('should return OK[200] if employee & business exists', async () => {
+            });
+            it('should return CONFLCIT[409] if employee or business does not exists', async () => {
+            });
+            it('should return BADREQUEST[400] if passed parameters are missing core fields.', async () => {
+            });
```
Comment (mth-1012) [inline src/api/controllers/tests/business-controller.test.ts:619]: put `// TODO: <something-here>` so we don't forget

File: src/api/controllers/tests/business-controller.test.ts [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+        describe('removeActivity endpoint', () => {
+            afterEach(() => {
+                server.close();
+            });
+            it('should return OK[200] if acitivty & business exists', async () => {
+            });
+            it('should return CONFLCIT[409] if activity or business does not exists', async () => {
+            });
+            it('should return BADREQUEST[400] if passed parameters are missing core fields.', async () => {
+            });
```
Comment (mth-1012) [inline src/api/controllers/tests/business-controller.test.ts:634]: put `// TODO: <something-here>` so we don't forget

File: src/api/controllers/tests/business-controller.test.ts [hunk 4]
```diff
@@ -0,0 +1,680 @@
...
+                server.close();
+            });
+            it('should return OK[200] if activity exists', async () => {
+            });
+            it('should return OK[200] if activity does not exists', async () => {
+            });
+            it('should return OK[200] if business does not exists', async () => {
+            });
+            it('should return BADREQUEST[400] if passed parameters are missing core fields.', async () => {
+            });
```
Comment (mth-1012) [inline src/api/controllers/tests/business-controller.test.ts:662]: put `// TODO: <something-here>` so we don't forget

File: src/api/controllers/tests/business-controller.test.ts [hunk 5]
```diff
@@ -0,0 +1,680 @@
...
+        describe('updateEmployee endpoint', () => {
+            afterEach(() => {
+                server.close();
+            });
+            it('should return OK[200] if employee exists', async () => {
+            });
+            it('should return OK[200] if employee does not exists', async () => {
+            });
+            it('should return BADREQUEST[400] if passed parameters are missing core fields.', async () => {
+            });
```
Comment (mth-1012) [inline src/api/controllers/tests/business-controller.test.ts:677]: put `// TODO: <something-here>` so we don't forget

**Review by mth-1012 — COMMENTED** [review]
Run `npm run prettier` and push again. Otherwise, looks good.

**Comment by p-carrier** [comment]
That's good you can merge @Mikeymousey

**Review by p-carrier — APPROVED** [review]

**Pull request merged**